app/extend/customTransport.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const FileTransport = require('egg-logger').FileTransport;
+const moment = require('moment');
+
+class CustomTransport extends FileTransport {
+  constructor(options, ctx) {
+    super(options);
+    this.ctx = ctx;
+  }
+
+  log(level, args, meta) {
+    const prefixStr = this.buildFormat(level);
+    for (let i in args) {
+      if (args.hasOwnProperty(i)) {
```

Comment:
The type of `args` is `any[]` https://github.com/eggjs/egg-logger/blob/master/index.d.ts#L197, so why not use `for...of` statement to iterate over `args`?